custom_components/sat/config_flow.py

```
@@ -671,6 +676,27 @@ async def async_step_presets(self, _user_input: dict[str, Any] | None = None):
             })
         )
 
+    async def async_step_areas(self, _user_input: dict[str, Any] | None = None):
+        room_weights = self._options.get(CONF_ROOM_WEIGHTS, {})
```

Ruff (0.8.2)

`CONF_ROOM_WEIGHTS` may be undefined, or defined from star imports

(F405)